index.js

```diff
@@ -175,10 +175,13 @@ module.exports = {
     
     "jsx-quotes": [2, "prefer-double"],
     "react/display-name": 0,
-    "react/jsx-boolean-value": 0,
+    "react/jsx-boolean-value": 2,
+    "react/jsx-closing-bracket-location": 2,
     "react/jsx-no-undef": 2,
+    "react/jsx-pascal-case": 2,
     "react/jsx-sort-prop-types": 0,
     "react/jsx-sort-props": 0,
+    "react/jsx-space-before-closing": 2,
```

Comment:
It's duplication of [this line](https://github.com/chimeraprime/eslint-config-chimera-prime/pull/2/files#diff-168726dbe96b3ce427e7fedce31bb0bcR198), isn't it?

Reply:
@Flixow Yes, I think so. It can be removed. @gh3r will do it soon.

Reply:
@Flixow Updated, we can now merge it